.gitlab-ci.yml

```diff
@@ -35,6 +35,22 @@ init:
     - schutzbot/update_github_status.sh start
   interruptible: true
 
+Regression:
+  stage: test
+  extends: .terraform
+  script:
+    - schutzbot/deploy.sh
+    - sudo schutzbot/selinux-context.sh
+    - sudo test/cases/regression_tests
+  artifacts:
+    when: always
+    paths:
+      - generated-image-infos/
+  parallel:
+    matrix:
+      - RUNNER:
+          - aws/fedora-35-x86_64
```

Comment:
Wouldn't it make more sense to use Fedora 36 instead? F35 will be EOL in November 2022, so it would save us some work for longer time.

Reply:
Not yet unfortunately. I need to land the updating proceedure that depends on this one. Because there's no image-info compiled for fedora-36 in the DB.
So first I need to land this PR, then the PR to update the DB. Then activate runners for fedora36, then update again the DB, then update the regression tests :)

Reply:
I'll open an issue here to remember to do that.

Reply:
What's the benefit of landing this now in that case? 🤔 Seems that it just creates more work for you.

Reply:
So, I want to be sure that next commits don't break anything. So I want at least to have one success build and one failed build for all the PR after this one. With the upcoming of automated updates, the image info generation is removed from the CI tests and only exist inside the triggered special pipelines to build them upon update of the manifests, which means that, there would be no test at all to ensure that nothing is broken upon a classic PR.

Reply:
So I kind of "need" that.